blockchains/bnb/bnb_worker.js

```diff
@@ -36,11 +35,12 @@ class MetricsStore {
 
 
 class BNBWorker extends BaseWorker {
-  constructor() {
-    super();
+  constructor(constants) {
```

Comment:
Provide the needed settings as constructor argument instead of the worker including them. This helps testing and makes more obvious what the worker needs.

Reply:
it might be more convenient if we call this `settings` instead of `constants`. `this.constants` sounds a bit weird.